components/datetime/tests/fixtures/tests/patterns.json

```diff
@@ -0,0 +1,24 @@
+[
```

Comment:
FYI: `patterns.bin` is empty.  Delete?

Reply:
It's a binary file, so it's showing up as empty in the github UI, but it 2.5kb.

<img width="608" alt="image" src="https://user-images.githubusercontent.com/1588648/111483685-0563a480-8703-11eb-9bde-d4c8bb912d42.png">

```
~/dev/icu4x on icu4x/components-bag at 09:28:07
➤ xxd ./components/datetime/tests/fixtures/tests/patterns.bin
00000000: 1600 0000 0000 0000 1800 0000 0000 0000  ................
00000010: 0100 0000 0000 0000 0000 0000 0200 0000  ................
00000020: 0000 0000 0000 0000 7000 0000 0000 0000  ........p.......
00000030: 0700 0000 0000 0000 0000 0000 0300 0000  ................
00000040: 0000 0000 0000 0000 0100 0000 0200 0000  ................
00000050: 0000 0000 2c20 0000 0000 0100 0000 0000  ...., ..........
00000060: 0000 0000 0000 0100 0000 0100 0000 0000  ................
```

It's used in `components/datetime/tests/pattern_serialization.rs`